python/GafferSceneUI/PathFilterUI.py

```diff
@@ -114,6 +118,56 @@
 
 )
 
+##########################################################################
+# VectorDataPlugValueWidget customisation
+###########################################################################
+
+class _PathsPlugValueWidget( GafferUI.VectorDataPlugValueWidget ) :
+
+	def __init__( self, plug, **kw ) :
+
+		GafferUI.VectorDataPlugValueWidget.__init__( self, plug, **kw )
+
+		self.vectorDataWidget().dataMenuSignal().connect( Gaffer.WeakMethod( self.__dataMenu ), scoped = False )
+
+	def __dataMenu( self, vectorDataWidget, menuDefinition ) :
+
+		selectedIndices = vectorDataWidget.selectedIndices()
+
+		if len( selectedIndices ) > 0 :
```

Comment:
I think we should show the menu item even when nothing is selected, but have it greyed out.

Reply:
Fixed in 3e96fded732a4d166ed6733e5b197d86d51ed308. As far as I can tell, it will probably not ever be disabled as right-clicking with no selection first selects the row. But this will cover that case should it ever change.